ironic/drivers/modules/drac/management.py

```diff
@@ -387,5 +400,77 @@ def known_good_state(self, task):
         node = task.node
 
         client = drac_common.get_drac_client(node)
+        if client.is_lifecycle_in_recovery():
+            LOG.info("Lifecycle Controller is in recovery mode")
```

Comment:
Since ironic may perform this operation on multiple nodes at once, I think we should add some identifier for the node in the above log message.

Reply:
Sure Chris.